helix-core/src/main/java/org/apache/helix/manager/zk/ZkBaseDataAccessor.java

```diff
@@ -61,6 +61,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.apache.helix.zookeeper.zkclient.ZkClient.DEFAULT_TTL;
```

Comment:
Usually we try to avoid static imports. ZkClient.DEFAULT_TTL should work?

Reply:
Thank you for the suggestion. Since I ended up breaking up the PR into multiple smaller ones, this variable doesn't show up in this PR anymore. But I'll import it the way you suggested in the next PRs.